client/os_dep.py

```diff
@@ -10,10 +10,11 @@
 
 def command(cmd):
     # this could use '/usr/bin/which', I suppose. But this seems simpler
-    for dir in os.environ['PATH'].split(':'):
-        file = os.path.join(dir, cmd)
-        if os.path.exists(file):
-            return file
+    if 'PATH' in os.environ:
+        for dir in os.environ['PATH'].split(':'):
+            file = os.path.join(dir, cmd)
+            if os.path.exists(file):
```

Comment:
@rbbratta, all the comments are valid, although the extra acess check wasn't so necessary given that autotest usually runs as root in the early beginnings of autotest. This code is pretty much ancient history and it wasn't updated since the change from tabs to spaces (Jun 6 2008).

So @cevich could write a newer version of this patch with your comments taken into account, if he's OK with it.